### DISCUSSION TITLE
PT-BR

### DISCUSSION THREAD

**Discussion by jepharaujo** [aclap-dev/video-downloadhelper#1081] [Language & Translation]
Here are the missing strings, translated to Brazilian Portuguese.

[pt_BR.txt](https://github.com/user-attachments/files/16464923/pt_BR.txt)

**Comment by D4rkF4lcon** [comment]
Here are the missing strings, translated to Brazilian Portuguese (18/08/2024).

[pt_BR.txt](https://github.com/user-attachments/files/16654551/pt_BR.txt)
